popup.js

```diff
@@ -12,12 +12,31 @@ function sendRequest(data) {
 }
 
 jQuery(document).ready(function($) {
-    //send message request to content script
-    $('#btnUpload').click(function() {
-        sendRequest({
-            type: 'addJson',
-            jsontext: $('#jsontext').val()
-        });
+    $('#form-import').submit(function(e) {
```

Comment:
@jinhduong Here is some other reasons:
- A form should handle with form's events :grin:
- When user doesn't want to use their mouse to click submit button (use Enter button is faster, that's why [Vim](http://www.vim.org/) is too popular).

My Facebook: [/manhtuan1412](http://m.me/manhtuan1412)